content/docs/querying/functions.md

```diff
@@ -256,6 +256,20 @@ label with the value `a` added to it:
 label_replace(up{job="api-server",service="a:c"}, "foo", "$1", "service", "(.*):.*")
 ```
 
+## `label_join()`
+
+For each timeseries in `v`, `label_join(v instant-vector, dst_label string, separator  string,
+src_label_1 string, src_label_2 string , ...)` joins all the values of all the `src_labels` 
+using `separator` and returns the timeseries with the label `dst_label` containing the joined value.
+If the final joined value is an empty string, label `dst_label` , if already present , is deleted.
```

Comment:
no space before comma.

I don't think there's a need to mention the empty string logic, that's the way all labels work.